test/e2e/platform_operators_test.go

```diff
@@ -135,14 +59,14 @@ var _ = Describe("platform operators controller", func() {
 					return bi.Spec.Template.Spec.ProvisionerClassName != bi.Spec.ProvisionerClassName
 				}).Should(BeTrue())
 			})
-			It("should choose the highest olm.bundle semver available in the catalog", func() {
-				Eventually(func() bool {
+			It("should choose the correct registry+v1 bundle image", func() {
+				Eventually(func() string {
 					bi := &rukpakv1alpha1.BundleDeployment{}
 					if err := c.Get(ctx, types.NamespacedName{Name: po.GetName()}, bi); err != nil {
-						return false
+						return ""
 					}
-					return bi.Spec.Template.Spec.Source.Image.Ref == "quay.io/operatorhubio/prometheus:v0.47.0"
-				}).Should(BeTrue())
+					return bi.Spec.Template.Spec.Source.Image.Ref
+				}).Should(ContainSubstring("registry.redhat.io/cert-manager/cert-manager-operator-bundle"))
```

Comment:
I favored this approach to avoid test brittleness over time. There's no guarantee that the redhat-operators catalog won't change the highest semver bundle between patch versions, so this should be more resilient to those kinds of changes.